### ISSUE TITLE
Key assignment in ReactJS branch

### ISSUE THREAD

**Issue by nickatomlin** [Designist/Korpus#16]
When using ReactJS, I'm consistently getting warnings about a missing "key" prop. For example: 
~~~~
"Warning: Each child in an array or iterator should have a unique "key" prop. Check the render method of `Sentence`. See https://fb.me/react-warning-keys for more information.
~~~~
I haven't seen any problems yet, but it would be nice to get rid of this bug. I'm currently getting this error for `Row`, `Sentence`, and `TimeBlock`.

**Comment by nickatomlin** [comment]
https://stackoverflow.com/a/26089858/6480948

**Comment by nickatomlin** [comment]
Fixed as of e953620577f7e650949d6740210a5521f9320fa5.
